keepassxc-browser/content/ui.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+// jQuery style wrapper for querySelector()
+var $ = function(elem)
```

Comment:
The scope is valid only between content scripts. It doesn't collide with jQuery or the page's JavaScript itself. In the current build `$` calls directly jQuery because jQuery is included in the content scripts.